editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -603,8 +603,7 @@ void EditorAssetLibrary::_notification(int p_what) {
 
 void EditorAssetLibrary::_update_repository_options() {
 	Dictionary default_urls;
-	default_urls["godotengine.org"] = "https://godotengine.org/asset-library/api";
-	default_urls["localhost"] = "http://127.0.0.1/asset-library/api";
+	default_urls["godotengine.org (Official)"] = "https://godotengine.org/asset-library/api";
```

Comment:
That shouldn't be in this commit, you probably squashed it in your instead of rebasing on top of `master`.